xml/System.IO/Directory.xml

```diff
@@ -158,74 +158,71 @@
+
+ 
+
+## Examples
+ The following example creates and deletes the specified directory.
+
  [!code-cpp[Dir_CreateDir#1](~/samples/snippets/cpp/VS_Snippets_CLR/Dir_CreateDir/CPP/dir_createdir.cpp#1)]
  [!code-csharp[Dir_CreateDir#1](~/samples/snippets/csharp/VS_Snippets_CLR/Dir_CreateDir/CS/dir_createdir.cs#1)]
- [!code-vb[Dir_CreateDir#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/Dir_CreateDir/VB/dir_createdir.vb#1)]  
-  
- To create the directory C:\Users\User1\Public\Html when the current directory is C:\Users\User1, use any of the following calls to ensure that the backslash is interpreted properly.  
-  
- In Visual Basic:  
-  
-```  
-Directory.CreateDirectory("Public\Html")   
-    Directory.CreateDirectory("\Users\User1\Public\Html")   
-    Directory.CreateDirectory("c:\Users\User1\Public\Html")  
-```  
-  
- In C#:  
-  
-```  
-Directory.CreateDirectory("Public\\Html");  
-    Directory.CreateDirectory("\\Users\\User1\\Public\\Html");  
-    Directory.CreateDirectory("c:\\Users\\User1\\Public\\Html");  
-```  
-  
- In C++:  
-  
-```  
-Directory::CreateDirectory("Public\\Html");  
-    Directory::CreateDirectory("\\Users\\User1\\Public\\Html");  
-    Directory::CreateDirectory("c:\\Users\\User1\\Public\\Html");  
-```  
-  
+ [!code-vb[Dir_CreateDir#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/Dir_CreateDir/VB/dir_createdir.vb#1)]
+
+ To create the directory C:\Users\User1\Public\Html when the current directory is C:\Users\User1, use any of the following calls to ensure that the backslash is interpreted properly.
+
+ In Visual Basic:
+
+```
+Directory.CreateDirectory("Public\Html") 
+    Directory.CreateDirectory("\Users\User1\Public\Html") 
+    Directory.CreateDirectory("c:\Users\User1\Public\Html")
+```
+
+ In C#:
+
+```
+Directory.CreateDirectory("Public\\Html");
+    Directory.CreateDirectory("\\Users\\User1\\Public\\Html");
+    Directory.CreateDirectory("c:\\Users\\User1\\Public\\Html");
+```
+
+ In C++:
+
+```
+Directory::CreateDirectory("Public\\Html");
+    Directory::CreateDirectory("\\Users\\User1\\Public\\Html");
+    Directory::CreateDirectory("c:\\Users\\User1\\Public\\Html");
+```
+
  ]]></format>
         </remarks>
-        <exception cref="T:System.IO.IOException">The directory specified by <paramref name="path" /> is a file.  
-  
- -or-  
-  
+        <exception cref="T:System.IO.IOException">The directory specified by <paramref name="path" /> is a file.
+
+ -or-
+
  The network name is not known.</exception>
         <exception cref="T:System.UnauthorizedAccessException">The caller does not have the required permission.</exception>
-        <exception cref="T:System.ArgumentException">
-          <paramref name="path" /> is a zero-length string, contains only white space, or contains one or more invalid characters. You can query for invalid characters by using the <see cref="M:System.IO.Path.GetInvalidPathChars" /> method.  
```

Comment:
The CI will change this again. Whenever the text starts with a tag, it will force to go to a new line.

Reply:
😢

Reply:
Ok, I reduced the number of changes @mairaw. Would you mind taking a look? Sorry for the mess!